peps/pep-0736.rst

```diff
@@ -0,0 +1,312 @@
+PEP: 736
+Title: Syntactic sugar for named arguments
+Author: Joshua Bambrick <jbambrick@google.com>,
+        Chris Angelico <rosuav@gmail.com>
+Sponsor: Guido van Rossum <guido@python.org>
+Discussions-To: https://discuss.python.org/t/syntactic-sugar-to-encourage-use-of-named-arguments/36217
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 28-Nov-2023
+Python-Version: 3.13
+Post-History: 14-Oct-2023
```

Comment:
Sorry for the late reply, I missed this at the time. 

https://peps.python.org/pep-0001/#discussing-a-PEP says:

> As soon as a PEP number has been assigned and the draft PEP is committed to the PEP repository, a discussion thread for the PEP should be created to provide a central place to discuss and review its contents, and the PEP should be updated so that the Discussions-To header links to it.

...

> If a PEP undergoes a significant re-write or other major, substantive changes to its proposed specification, a new thread should typically be created in the chosen venue to solicit additional feedback. If this occurs, the Discussions-To link must be updated and a new Post-History entry added pointing to this new thread.